packages/api/es/search.js

```diff
@@ -34,24 +34,31 @@ const getCredentials = () =>
  * @returns {string} elasticsearch local address
  */
 const getLocalEsHost = () => {
-  if (process.env.LOCAL_ES_HOST) return `${process.env.LOCAL_ES_HOST}:9200`;
-  if (process.env.LOCALSTACK_HOST) return `${process.env.LOCALSTACK_HOST}:4571`;
+  if (process.env.LOCAL_ES_HOST) return `http://${process.env.LOCAL_ES_HOST}:9200`;
+  if (process.env.LOCALSTACK_HOST) return `http://${process.env.LOCALSTACK_HOST}:4571`;
   return 'localhost:9200';
 };
 
 const esTestConfig = () => ({
-  host: getLocalEsHost(),
+  node: getLocalEsHost(),
   requestTimeout: 5000
 });
 
 const esProdConfig = async (host) => {
   if (!aws.config.credentials) await getCredentials();
 
+  let node = 'http://localhost:9200';
+
+  if (process.env.ES_HOST) {
+    node = `http://${process.env.ES_HOST}`;
```

Comment:
For our lambdas, won't the ES host address be on `https`?

Reply:
Hmmm good question. I did a ton of testing on this and `http` seemed to work fine. I'll try with `https` just to see.

Reply:
I changed it to `https`, but it seems to work either way due to the security group allowing all traffic within the VPC. Both ES and lambda are behind the VPC.